Tests/Common/TestEui.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Common
+{
+    using System.Security.Cryptography;
+
+    public static class TestEui
+    {
+        public static DevEui GenerateDevEui() => new DevEui((uint)RandomNumberGenerator.GetInt32(int.MaxValue));
```

Comment:
I would start from 1 so it never generates an invalid DevEUI and also remark that only the lower 31 bits are set:

```suggestion
        /// <remarks>
        /// While DevEUI are 64 bits wide, this generates a random DevEUI with only 31 LSB set.
        /// </remarks>
        public static DevEui GenerateDevEui() => new(unchecked((uint)RandomNumberGenerator.GetInt32(int.MaxValue) + 1));
```